lib/debt_to_income_ratio_latest/main.py

```diff
@@ -96,25 +103,25 @@ async def get_debt_to_income_ratio_latest(
     if len(depository_credit_records_list) == 0:
         return float("inf")
 
-    # Here we sort by timestamp so latest balances are on top
+    # STEP 6: Here we sort by timestamp so latest balances are on top
     loan_records_list = sorted(loan_records_list, key=lambda x: x["ts"], reverse=True)
 
-    # Then we loop through all balances per institution and account and store the latest balance
+    # STEP 7: Then we loop through all balances per institution and account and store the latest balance
     latest_balances = {}
     for loan_record in loan_records_list:
         key = (loan_record["institution_id"], loan_record["account_id"])
         if key not in latest_balances:
             latest_balances[key] = loan_record["balance"]
     
-    # Finally, we can sum all the balances
+    # STEP 8: Sum all the balances
     sum_of_loan_balances_latest = sum(latest_balances.values())
 
-    # Sum of credit transactions across all depository accounts
+    # STEP 9: Sum of credit transactions across all depository accounts
     sum_of_depository_credit_transactions = 0
     for credit in depository_credit_records_list:
         sum_of_depository_credit_transactions += credit["amount"]
 
-    # Compute debt to income ratio
+    # STEP 10: Compute debt to income ratio
     ratio = sum_of_loan_balances_latest / sum_of_depository_credit_transactions
```

Comment:
Is it ever possible that `sum_of_depository_credit_transactions` could be 0 and make this throw an error?

Reply:
Totally. Will patch this.

Reply:
I checked and in this case we are ensuring that the list `depository_credit_records_list` has items. If not, it returns `inf`earlier in the code.

I will add here a comment explaining why this is not a problem at this point in the code.